src/pilot/templates/nic-configs/dvr_7_port/compute.yaml

```diff
@@ -0,0 +1,230 @@
+    default: ''
+    description: IP address/subnet on the external network
+    type: string
+  InternalApiIpSubnet:
+    default: ''
+    description: IP address/subnet on the internal API network
+    type: string
+  StorageIpSubnet:
+    default: ''
+    description: IP address/subnet on the storage network
+    type: string
+  StorageMgmtIpSubnet:
+    default: ''
+    description: IP address/subnet on the storage mgmt network
+    type: string
+  TenantIpSubnet:
+    default: ''
+    description: IP address/subnet on the tenant network
+    type: string
+  ManagementIpSubnet: # Only populated when including environments/network-management.yaml
+    default: ''
+    description: IP address/subnet on the management network
+    type: string
+  ExternalNetworkVlanID:
+    default: 10
+    description: Vlan ID for the external network traffic.
+    type: number
+  InternalApiNetworkVlanID:
+    default: 20
+    description: Vlan ID for the internal_api network traffic.
+    type: number
+  StorageNetworkVlanID:
+    default: 30
+    description: Vlan ID for the storage network traffic.
+    type: number
+  TenantNetworkVlanID:
+    default: 50
+    description: Vlan ID for the tenant network traffic.
+    type: number
+  ManagementNetworkVlanID:
+    default: 60
+    description: Vlan ID for the management network traffic.
+    type: number
+  ControlPlaneSubnetCidr: # Override this via parameter_defaults
+    default: '24'
+    description: The subnet CIDR of the control plane network.
+    type: string
+  ControlPlaneDefaultRoute: # Override this via parameter_defaults
+    description: The default route of the control plane network.
+    type: string
+  ExternalInterfaceDefaultRoute: # Not used by default in this template
+    default: '10.0.0.1'
+    description: The default route of the external network.
+    type: string
+  ManagementInterfaceDefaultRoute: # Commented out by default in this template
+    default: unset
+    description: The default route of the management network.
+    type: string
+  DnsServers: # Override this via parameter_defaults
+    default: []
+    description: A list of DNS servers (2 max for some implementations) that will be added to resolv.conf.
+    type: comma_delimited_list
+  EC2MetadataIp: # Override this via parameter_defaults
+    description: The IP address of the EC2 metadata server.
+    type: string
+  ComputeProvisioningInterface:
+    default: 'em3'
+    description: Provisioning interface name
+    type: string
+  ComputeBond0Interface1:
+    default: 'em1'
+    description: Bond 0 interface 1 name
+    type: string
+  ComputeBond0Interface2:
+    default: 'p3p1'
+    description: Bond 0 interface 2 name
+    type: string
+  ComputeBond1Interface1:
+    default: 'em2'
+    description: Bond 1 interface 1 name
+    type: string
+  ComputeBond1Interface2:
+    default: 'p3p2'
+    description: Bond 1 interface 2 name
+    type: string
+  ComputeBond2Interface1:
+    default: 'p1p1'
```

Comment:
Struggling to understand why we would add 2 1 port NICs instead of a single dual port NIC that's identical to what's already in the server.

Reply:
They are two port NICs, only that two interfaces (p2p2 & p3p2) aren't used. Basically this was done to ensure high availability at NIC level. There is another option that does this with three NICs and without affecting the NIC level HA. 

Bond0: em1 & p1p1
Bond1: em2 & p2p1
Bond2: p1p2 & p2p2